cluster/gce/gci/configure-helper.sh

```diff
@@ -25,6 +25,27 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function convert-params-array-to-yaml {
+  # A helper function to convert the manifest args from a string to a yaml list
+  # of arguments.
+  # Old format:
+  #   command=["/bin/sh", "-c", "exec KUBE_EXEC_BINARY --param1=val1 --param2-val2"].
+  # New format:
+  #    command:
+  #    - KUBE_EXEC_BINARY  # No shell dependencies.
+  #    args:
+  #    - --param1=val1
+  #    - --param2-val2
+  IFS=' ' read -ra PARAMS <<< "$@"
+  flags=""
+  for flag in "${PARAMS[@]}"; do
+    flags+="    - $flag\n"
```

Comment:
Given the indentation, this is only useable for static pods, so there should be a big disclaimer.

Reply:
+1